rpcs3/Emu/Cell/lv2/sys_usbd.cpp

```diff
@@ -522,10 +522,11 @@ usb_handler_thread::usb_handler_thread()
 		}
 	}
 
-	for (const auto& [entry, count] : emulate_device_check)
+	for (const auto& entry : device_allow_list)
 	{
 		if (entry.max_device_count && entry.make_instance)
 		{
+			auto count = emulate_device_check[entry];
```

Comment:
Honestly, the name of the variable doesn't really tell me much.
Maybe rename it to something that's easier to understand.
```suggestion
			const int count = emulate_device_check[entry];
```